gammapy/image/basic.py

```diff
@@ -260,26 +263,69 @@ def _counts(self, observation):
         counts.fill_events(events)
         return counts
 
+    def _acceptance_map(self, observation, counts, nbins=10):
+        """Compute acceptance (normalized background map
+
+        Parameters
+        ----------
+        observation : `~gammapy.data.DataStoreObservation`
+            Observation object
+        counts : `~gammapy.image.SkyImage`
+            count sky images
+        nbins : int
+            number of bins for the energy integration
+        """
+        p = self.parameters
+        ebins = EnergyBounds.equal_log_spacing(p['emin'], p['emax'], nbins)
+        acceptance = SkyImage.empty_like(counts)
+        acceptance.name = 'exposure_on'
+
+        offsets = observation.pointing_radec.separation(acceptance.coordinates())
+
+        # This is a somewhat dirty approach to deal with the different background IRFs
```

Comment:
You could just add a uniform evaluate on the two background classes for now as a new method, with any name and signature you like. We will have to work on IRF classes for weeks / months, I don't think we should be shy to edit them, we will have to iterate there, and anyways IRFs should be considered low-level, not super stable API for now that most users don't use directly.

Reply:
I suppose we won't be supporting EnergyOffsetArray for long so I am not sure it is worth changing the API for this one, no?

Reply:
Agreed!